sci-biology/qrna/qrna-2.0.3c-r2.ebuild

```diff
@@ -16,7 +16,7 @@ IUSE=""
 
 RDEPEND="
 	dev-lang/perl
-	sci-biology/hmmer"
+	sci-biology/hmmer:2"
```

Comment:
Thanks, this will require a revbump, so please bump this to -r3 and remove -r2 in a separate commit.

Reply:
Done. BTW, what would be wrong with `git mv qrna-2.0.3c-r2.ebuild qrna-2.0.3c-r3.ebuild`?

Reply:
In general, easy ability to revert atomic changes. https://wiki.gentoo.org/wiki/Gentoo_GitHub#Separate_.28or_.22atomic.22.29_commits